src/Microsoft.AspNetCore.Antiforgery/ServiceCollectionExtensions.cs

```diff
@@ -10,9 +10,16 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for setting up antiforgery services in an <see cref="IServiceCollection" />.
+    /// </summary>
     public static class ServiceCollectionExtensions
     {
-        public static IServiceCollection AddAntiforgery(this IServiceCollection services)
+        /// <summary>
+        /// Adds antiforgery services to the specified <see cref="IServiceCollection" />.
```

Comment:
After [discussing](https://github.com/aspnet/Mvc/pull/4159#discussion-diff-54208453) this point in aspnet/Mvc#4159, I agree completely. @javiercn should leave this as-is.